src/app/pages/BuyPage/components/SwapAssetSelector/index.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
I think we can use one of `app/components/AssetSelect` or `app/components/Form/AssetSelect` without creating new components.